examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/speech_client_config.json

```diff
@@ -0,0 +1,41 @@
+{
+  "interfaces": {
+    "google.cloud.speech.v1.Speech": {
+      "retry_codes": {
```

Comment:
gRPC is eventually going to support client retries natively, at which point we will want to drop our implementation. It'll be a least a year until we see it though. 

However, we did decide to drop retry from the configuration for this reason and we won't be able to generate this file as we do today. What we can do in the interm is generate this file with a fixed retry for methods that are annotated as `get` methods and then have synth perform any customizations. Thoughts?